### ISSUE TITLE
Eliminate dot macro

### ISSUE THREAD

**Issue by travigd** [JuliaGizmos/Blink.jl#240]
I'd like to eliminate the `@dot` and `@dot_` macros (I'd like to eliminate the underscore-postfixed macros anyway in favor of a keyword argument but that's for later).

Truthfully, I think the macro isn't very self-explanatory. It confused me for quite some time (probably longer than it should have, truthfully). I'd rather allow interpolation of windows into JSExpr expressions (this shouldn't be too hard to do, especially with JSExpr/WebIO v1.0) so that we'd do something like this. It's also probably logical to execute JavaScript with `this` bound to the receiver object. 

```julia
focus!(window::Window) = @js window $window.focus()
focus!(window::Window) = @js window this.focus()
```

This also makes it easier for things to be implemented in "userspace" (since `@dot` is unexported).

**Comment by NHDaly** [comment]
Yeah makes sense to me. A lot of the Javascript interaction is confusing. The difference between `@js` and `@js_` as well, yeah.

LGTM
